.github/workflows/cargo_bloat.yml

```diff
@@ -0,0 +1,83 @@
+name: Binary size report
+on:
+  push:
+    branches: [ develop ]
+    paths:
+      - 'libraries/**/*.rs'
+      - 'src/**/*.rs'
+      - 'patches/**'
+      - '**/Cargo.toml'
+      - '.cargo/config'
+      - '!third_party/**'
+  pull_request:
+    branches: [ develop ]
+    types: [opened, synchronize, reopened]
```

Comment:
I'm not convinced about this whole "on" specification. There are open questions like:
- Do we want to restrict to those paths? Why not run without "paths" restriction.
- Do we want to restrict to develop? Why not run on stable too.
- Why do we need to specify the default for `pull_request.types`? (see first note [here](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request))

But I'm fine to keep like that and update when we have an error. But I usually prefer to do the opposite. Keep the initial specification as broad as possible or as close to the default as possible.

Reply:
Simplified to just pull_request.